service/debugger/debugger.go

```diff
@@ -967,6 +967,19 @@ func (d *Debugger) FindThread(id int) (proc.Thread, error) {
 	return nil, nil
 }
 
+// FindGoroutine returns the goroutine for the given 'id'.
+func (d *Debugger) FindGoroutine(id int) (*proc.G, error) {
+	d.targetMutex.Lock()
+	defer d.targetMutex.Unlock()
+
+	g, err := proc.FindGoroutine(d.target, id)
```

Comment:
Just `return proc.FindGoroutine(...)` is enough here.

Reply:
Done.